CONTRIBUTING.md

```diff
@@ -291,19 +291,30 @@ In some cases, it may be necessary for a test to intentionally violate the rules
 
 ## Procedurally-generated tests
 
+There are two systems for procedurally generating tests. To generate/regenerate all generated tests, run these two commands:
+
+```
+./make.py
+./tools/generation/generate_type_coercion.py
+```
```

Comment:
in a follow up, this could be done as anchor links to each generation topic, so it's easier to understand why each one is running.